python/py-tensorflow1/Portfile

```diff
@@ -48,21 +50,24 @@ patch.pre_args      -p1
 
 # Required java version
 java.version        11+
-# JDK port to install if required java not found
-java.fallback       openjdk12
+# LTS JDK port to install if required java not found
+java.fallback       openjdk11
 # JDK only needed at build time, but java PG sets lib dependency so
 # declare no conflict to allow redistribution of binaries.
-license_noconflict  openjdk12
+license_noconflict  ${java.fallback}
```

Comment:
Public updates for Java 12 ended 2019-09. Java 11 is the latest LTS version (public updates until 2024-10 for OpenJDK); I suggest an LTS version be used as ports' fallback to minimize the frequency of having to update portfiles.